src/ontology/bridge/uberon-bridge-to-mba.obo

```diff
@@ -934,8 +941,12 @@ property_value: status "Verified" xsd:string {source="https://ror.org/03cpe7c52"
 [Term]
 id: MBA:1079
 intersection_of: UBERON:0002985
+intersection_of: UBERON:8440077
+intersection_of: BFO:0000050 NCBITaxon:10090
 intersection_of: BFO:0000050 NCBITaxon:10090
```

Comment:
Why the duplicated `intersection_of` tag? I don’t think it does any harm as the redundant tag seems ignored by the OBO parser, but still, probably better to avoid that?

Reply:
this one is strange - wonder if its me duplicating some equiv stuff with different annotations on it that the obo parser is confused? - A lot of the mappings are manual which = duplications >.< well anyway its ok for now - once I get the final look through by SME, and given the thumbs up, I'll clean stuff up :) Thanks @gouttegd